src/map/status.cpp

```diff
@@ -802,6 +802,10 @@ void initChangeTables(void)
 	set_sc( GD_BATTLEORDER		, SC_BATTLEORDERS	, EFST_GDSKILL_BATTLEORDER	, SCB_STR|SCB_INT|SCB_DEX );
 	set_sc( GD_REGENERATION		, SC_REGENERATION	, EFST_GDSKILL_REGENERATION	, SCB_REGEN );
 
+#ifdef RENEWAL
```

Comment:
Extra `#ifdef RENEWAL` ?

Reply:
I added this to avoid any warnings in skill checks for pre-renewal. This makes it so we don't have to define the skill in pre-re.